### ISSUE TITLE
Fix Tifffile version to 2019 for imctools v1

### ISSUE THREAD

**Issue by votti** [BodenmillerGroup/imctools#85]
The latest `tifffile` version breaks imctools v1 as the description cannot be written in ome.tiff

This is as `tifffile` now started writing their own `ome.tiff` writer.

Thus I propose to make a new release of imctools v1 where the tiffile version is fixed to `2019.7.26`.

```
2020.7.17
    Pass 3022 tests. Initial support for writing OME-TIFF (WIP). Return samples as separate dimension in OME series (breaking). Fix modulo dimensions for multiple OME series. Fix some test errors on big endian systems (#18). Fix BytesWarning. Allow to pass TIFF.PREDICTOR values to TiffWriter.save.
```

**Comment by cgohlke** [comment]
>  the description cannot be written in ome.tiff

this should be fixed in tifffile-2020.7.22

**Comment by plankter** [comment]
Fixed in imctools v1.0.8
